packages/commonwealth/client/scripts/views/components/UserSurveyGrowl/UserSurveyGrowl.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import useGrowlStore from 'state/ui/growl';
+import { CWCheckbox } from 'views/components/component_kit/cw_checkbox';
+import { CWGrowl } from 'views/components/component_kit/cw_growl';
+import { CWIconButton } from 'views/components/component_kit/cw_icon_button';
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWButton } from '../component_kit/new_designs/CWButton';
+import './UserSurveyGrowl.scss';
+
+const LOCALSTORAGE_USER_SURVEY_GROWL_KEY = 'stakeGrowlHidden';
```

Comment:
lets use a different key, we have used `stakeGrowlHidden` before, so some users might still have it in their localstorage